lib/modules/apostrophe-events-calendar-widgets/public/css/calendar.less

```diff
@@ -0,0 +1,135 @@
+@white: white;
+@black: #000F3A;
+@blue-darkest: #000F3A;
+@blue-dark: #00144D;
+@blue-medium: #01256E;
+@blue-light: #045EA7;
+@blue-lightest: #82AFD3;
+@blue-transparent: rgba(130, 175, 211, 0.15);
+
+@red-dark: #57000A;
+@red: #95001A;
+
+@gray: #EFEFEF;
+@gray-fade: #2c2f39;
+
+.c-calendar {
+  background-color: @white;
+  padding: 2rem 1rem;
+  margin-bottom: 2rem;
+  font-size: 16px;
+}
+
+.c-cal-month {
+  display: block;
+  text-align: center;
+}
+
+.c-cal-grid {
+  display: flex;
```

Comment:
I don't think we have autoprefixing on this, so these all need to be prefixed.

Reply:
ran autoprefixer-cli on this file